site-scrapers/CVS.js

```diff
@@ -0,0 +1,50 @@
+const sites = require("../data/sites.json");
+const https = require("https");
+const crypto = require("crypto");
+const { map } = require(".");
+
+const siteName = "CVS";
+const site = sites[siteName];
+
+module.exports = async function GetAvailableAppointments(browser) {
+    console.log(`${siteName} starting.`);
+    const webData = await ScrapeWebsiteData(browser);
+    console.log(`${siteName} done.`);
+    return webData.map((responseLocation) => {
+        let hasAvailability = parseInt(responseLocation.totalAvailable)
```

Comment:
Always use a radix with `parseInt`, i.e. `parseInt(responseLocation.totalAvailable, 10)`

Reply:
Per our convo, it should default to 10. Octal is depreciated.